### DISCUSSION TITLE
مشکل در اجرای تاریخ و زمان

### DISCUSSION THREAD

**Discussion by Saeedr** [majidh1/JalaliDatePicker#84] [Question]
روی ساعت هم وقتی روی مدل هست سلکت آپشن که باز میشه سریع بسته میشه . اگر تلگرام داری یک پیام بده برات فیلمشو بزارم . 
@Saeedrajayi

https://github.com/majidh1/JalaliDatePicker/assets/7427773/4e9903a8-f312-431e-aa4e-4da6f57696c8

![Screenshot 2024-02-26 121509](https://github.com/majidh1/JalaliDatePicker/assets/7427773/9bf6435f-dc4c-4ba3-b283-0a15bb2a8d49)

**Comment by majidh1** [comment]
میتونی یه لینک بدی تست کنم روی jsfiddle  یا codepen

**Reply by Saeedr** [reply]
نه متاسفانه ولی می تونم بهتون انی دسک بدم بهم متصل بشید SaeedRajayi@ تلگرام یا 09156000647

**Comment by majidh1** [comment]
باگ عجیبیه و فکر میکنم که از سمت خود فایرفاکس باشه به هر حال با حذف tabindex از modal توی تست من درست شد
میتونید نمونه رو اینجا چک کنید
https://jsfiddle.net/gktbds5L/

**Reply by Saeedr** [reply]
اصلا این کار نمی کنه متوجه نمیشم می خوام فقط ساعت نشون بدم انگار تو باغ نیستم .
      <a href="#" class="btn btn-lg btn-success" data-toggle="modal" data-target="#basicModal">Click to open Modal Fix</a>

      <a href="#" class="btn btn-lg btn-danger" data-toggle="modal" data-target="#basicModal2">Click to open Modal Bug</a>

<div class="modal fade" id="basicModal">
  <div class="modal-dialog" tabindex="-1">
    <div class="modal-content p-5">
      Fixed:
      <input data-jdp-only-time placeholder="JalaliDatePicker">
    </div>
  </div>
</div>

<div class="modal fade" id="basicModal2" tabindex="-1">
  <div class="modal-dialog">
    <div class="modal-content p-5">
      Buged:
      <input data-jdp-only-time placeholder="JalaliDatePicker">
    </div>
  </div>
</div>
